src/main/java/com/powsybl/openloadflow/network/PiModelArray.java

```diff
@@ -101,6 +101,14 @@ public double getB2() {
         return getModel().getB2();
     }
 
+    public double getModifiedR1() {
```

Comment:
Why do you need these two methods? It works with `getR1()` and `getA1()`.

Reply:
I need to raw value of a1 and r1 to save it only if overridden. Look at getA1 and getR1, in case of nan it return value at tap position.

Reply:
Maybe "continuous" instead of "modified"?

Reply:
We already have a continuousR1 to store the continuous value of R1 even after rounding